root/components/FormRowSelectList.js

```diff
@@ -19,6 +19,7 @@ type Props<S> = {|
   +addId: string,
   +addLabel: string,
   +getSelectField: (S) => FieldT<number | string>,
+  +hideAddButton: boolean,
```

Comment:
Hiding add button should be optional.

```suggestion
  +hideAddButton?: boolean,
```

Reply:
Fixed.